mockgcp/tools/gapic/pkg/protogen/convert.go

```diff
@@ -669,7 +675,7 @@ func (c *OpenAPIConverter) resolveMessageType(ref string) string {
 	}
 
 	switch ref {
-	case "Operation":
+	case "Operation", "GoogleLongrunningOperation":
```

Comment:
it's a little bit hard to trace here. May I ask what the issues are (guess that may be specific to the api from raw json)  ?

Reply:
So this is converting an OpenAPI schema to a proto.  That isn't a perfect operation - we have lost some information - not least the proto tag numbers.  We want to generate the right type for an LRO, and this is how LROs show up (at least for this OpenAPI)